code-challenges.test.js

```diff
@@ -13,27 +15,68 @@
 // --------------------INSTRUCTOR EXAMPLE: Create a function that takes in an array of numbers and returns an array with all the numbers multiplied by 3.
 
 // a) Create a test with expect statements for each of the variables provided.
-
 const numbersArray1 = [6, 7, 8, 9, 10]
 // Expected output: [18, 21, 24, 27, 30]
+
 const numbersArray2 = [24, 27, 30, 33, 36]
 // Expected output: [72, 81, 90, 99, 108]
 
 // b) Create the function that makes the test pass.
 
+// -------------------------<break>---------------------------------
+
+// Jest template
+// describe (" ", () => {
+//   it (" ", () => {
+//     expect().toEqual()
+//   })
+// })
+
 // --------------------1) Create a function that takes a object as an argument and decides if the number inside it is evenly divisible by three or not.
 
 // a) Create a test with expect statements for each of the variables provided.
 
-const object1 = { number: 15 }
-// Expected output: "15 is divisible by three"
-const object2 = { number: 0 }
-// Expected output: "0 is divisible by three"
-const object3 = { number: -7 }
-// Expected output: "-7 is not divisible by three"
+describe ("divisibleByThree", () => {
+    it ("takes a object as an argument and decides if the number inside it is evenly divisible by three or not.", () => {
+      expect(divisibleByThree(15)).toEqual("is divisible by three")
+      expect(divisibleByThree(0)).toEqual("is divisible by three")
+      expect(divisibleByThree(-7)).toEqual("is not divisible by three")
```

Comment:
I would recommend passing the test variables into your test instead of a hard-coded number.  The test variables are not numbers, but they are objects and will need to be handled differently in your function as a result.